threepenny-gui.cabal

```diff
@@ -119,6 +119,7 @@ Library
                     ,file-embed             == 0.0.10
                     ,hashable               >= 1.1.0 && < 1.3
                     ,safe                   == 0.3.*
+                    ,time                   >= 1.6.1 && < 1.9
```

Comment:
I would like to have `time >= 1.4.*` here, to ensure compatibility with older GHC versions.
We have CI in place to test whether they compile.